modules/react/tabs/lib/TabsList.tsx

```diff
@@ -46,44 +44,90 @@ export const useTabsList = composeHooks(
   useListResetCursorOnBlur
 );
 
-const StyledStack = styled(Flex, {
-  shouldForwardProp: filterOutProps(['maskImage']),
-})<StyledType & {maskImage?: string}>(({maskImage}) => ({
-  maskImage: maskImage,
-}));
+const tabsListStencil = createStencil({
```

Comment:
```suggestion
export const tabsListStencil = createStencil({
```

Reply:
this line can probably be added to the stencil instead; https://github.com/Workday/canvas-kit/pull/3119/files#diff-5e3729c5f87ee54b9d212a45937aac7b46d5314877e6b5b1e8182b7d55a5901dL42

We should remove styles in model hooks so the css makes sense.